Code/GraphMol/MolDiscriminators.cpp

```diff
@@ -57,7 +57,7 @@ double computeBalabanJ(const ROMol &mol, bool useBO, bool force,
                        const std::vector<int> *bondPath, bool cacheIt) {
   RDUNUSED_PARAM(useBO);
   double res = 0.0;
-  if (!force && mol.hasProp(common_properties::BalanbanJ)) {
+  if (!force && mol.hasProp(common_properties::BalabanJ)) {
```

Comment:
This is a really, really old bug fix from a string typo "Balana_**n**_bJ" versus "BalabanJ".  Didn't hurt anything though!